.github/workflows/pr-backend-unit-tests-enterprise.yml

```diff
@@ -41,42 +42,47 @@ jobs:
           sudo apt-get install -y build-essential shared-mime-info
           go install github.com/mfridman/tparse@c1754a1f484ac5cd422697b0fec635177ddc8507 # v0.17.0
 
-      - name: Verify code generation
-        run: |
-          CODEGEN_VERIFY=1 make gen-cue
-          CODEGEN_VERIFY=1 make gen-jsonnet
+      - name: Setup Enterprise
+        uses: ./.github/actions/setup-enterprise
+        with:
+          github-app-name: 'grafana-ci-bot'
 
       - name: Generate Go code
         run: make gen-go
+        env:
+          WIRE_TAGS: 'enterprise'
 
       - name: Run unit tests
-        run: COVER_OPTS="-coverprofile=be-unit.cov -coverpkg=github.com/grafana/grafana/..." GO_TEST_OUTPUT="/tmp/unit.log" make test-go-unit
-
-      - name: Process and upload coverage
-        uses: ./.github/actions/test-coverage-processor
-        with:
-          test-type: 'be-unit'
-          # Needs to be named 'unit.cov' based on the Makefile command `make test-go-unit`
-          coverage-file: 'unit.cov'
-          codecov-token: ${{ secrets.CODECOV_TOKEN }}
-          codecov-flag: 'be-unit'
-          codecov-name: 'be-unit'
+        run: |
+          echo "test backend unit tests with coverpkg"
+          GO_TEST_FILES=$(./scripts/go-workspace/test-includes.sh)
+          go test -race -short -covermode=atomic -coverprofile=${{ env.TEST_TYPE }}.cov -coverpkg=github.com/grafana/grafana/... -timeout=30m ${GO_TEST_FILES} -json | tee /tmp/${{ env.TEST_TYPE }}.log | tparse -all
+        env:
+          WIRE_TAGS: 'enterprise'
 
       - name: Install Grafana Bench
-        # We can't allow forks here, as we need secret access.
-        if: ${{ github.event_name != 'pull_request' }}
         uses: ./.github/actions/setup-grafana-bench
+        with:
+          github-app-name: 'grafana-ci-bot'
 
       - name: Process output for Bench
-        if: ${{ github.event_name != 'pull_request' }}
         run: |
           grafana-bench report \
             --trigger pr-backend-unit-tests-oss \
             --report-input go \
             --report-output log \
             --grafana-version "$(git rev-parse HEAD)" \
-            --suite-name grafana-oss-unit-tests \
-            /tmp/unit.log
+            --suite-name grafana-oss-and-enterprise-unit-tests \
```

Comment:
@Proximyst does this naming convention jive with your file and locking schema?

Reply:
Sure! It's just used for filtering in Loki